requirements/requirements.txt

```diff
@@ -14,7 +14,6 @@ justext==3.0.1
 lxml_html_clean
 mwparserfromhell==0.6.5
 nemo_toolkit[nlp]>=1.23.0
-numpy<2
```

Comment:
Do you just want to have `numpy` without any pin instead of removing it entirely?